paper-radio-button.html

```diff
@@ -114,6 +114,7 @@
     },
 
     ready: function() {
+      this.toggles = true;
```

Comment:
Is this a property inherited from `Polymer.PaperRadioButtonBehavior`? Is it possible to override the default in the `properties` block instead?